dev/lint-python

```diff
@@ -104,7 +104,7 @@ function mypy_data_test {
       -c dev/pyproject.toml \
       --rootdir python \
       --mypy-only-local-stub \
-      --mypy-ini-file python/mypy.ini \
+      --mypy-ini-file dev/mypy.ini \
```

Comment:
Okay, MyPy requires the configuration file to locate in the root directory of the package. Let me move it back for now.